srcpkgs/opensmtpd-filter-senderscore/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'opensmtpd-filter-senderscore'
+pkgname=opensmtpd-filter-senderscore
+version=0.1.1
+revision=1
+archs="x86_64* i686*"
```

Comment:
why is it restricted to these architectures?

Reply:
When I created this pull request, the CI thing failed for all architectures except those two. Since I don't have any of the affected hardware, I can't figure out why.

Reply:
You can cross compile for these architectures (for example `./xbps-src -a aarch64 pkg ...`).
After rebasing on current master, a relatively simple fix in do_install should do it.

Reply:
It seems to work for all archs now.